src/Atmos/Model/boundaryconditions.jl

```diff
@@ -55,6 +61,11 @@ function boundary_state!(
     t,
     args...,
 )
+
+    ntargs = (; n, state = state⁻, aux = aux⁻, t, args)
```

Comment:
I’m thinking it might be helpful to add a suffix to `n`: outward, inward, downward or upward (I need to double check which one)